src/dags/importscripts/import_processenverbaalverkiezingen.py

```diff
@@ -0,0 +1,134 @@
+import csv
+import ftplib
+from more_ds.network.url import URL
+from os import path as ospath
+from typing import List, TypedDict, Tuple, Optional, Generator
+
+
+class Data(TypedDict):
+    """Datastructure processenverbaal verkiezingen"""
+
+    verkiezingsjaar: int
+    volgnummer: str
+    uri: str
+    documentnaam: str
+    stemlocatie: str
+    id: str
+
+
+class FTPListing:
```

Comment:
Is this using a special FTP server? Or just the objectstore using the FTP protocol?

Reply:
You are right :+1:  This is indeed the Objectstore. To me more precise I will rename it to ObjectStoreListing